dvc/command/diff.py

```diff
@@ -134,17 +133,17 @@ def add_parser(subparsers, parent_parser):
     diff_parser.add_argument(
         "a_ref",
         help=(
-            "Git reference to the old version that you want to compare"
-            " (defaults to HEAD)"
+            "Git reference to the older version to compare"
+            " (defaults to `HEAD`)"
         ),
         nargs="?",
         default="HEAD",
     )
     diff_parser.add_argument(
         "b_ref",
         help=(
-            "Git reference to the new version that you want to compare."
-            " (defaults to the working tree)"
+            "Git reference to the newer version to compare"
+            " (defaults to the current workspace including changes)"
```

Comment:
what do you mean by _including changes_? :thinking:

Reply:
Just trying to be extra clear that the workspace includes any uncommitted local changes. I guess its confusing, we could just leave "current workspace" which implies the rest.

Reply:
@jorgeorpinel , yep, it was confusing to me, tbh; I appreciate the rest of the changes, tho :+1:

Reply:
Simplified in 6984546a.